test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -441,6 +441,16 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $result.Error.Exception.Message | Should Match ": 418 \(I'm a teapot\)\."
         $result.Error.FullyQualifiedErrorId | Should Be "WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand"
     }
+
+    It "Validate Invoke-WebRequest returns HTTPS errors in exception" {
```

Comment:
Maybe  make this more accurately:
"Validate Invoke-WebRequest returns native HTTPS error messages in exception"
And the same in test below.